queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals
+WHERE name LIKE '%mon%';
```

Comment:
- [ ] This query finds all animals its name has `"mon"` in any position, however, we're asked to find all animals that its name ends in `"mon"`. Kindly remove the last `%` from the query so you will have;

```ruby
   SELECT * from animals
  WHERE name LIKE '%mon';
```